packages/money-legos/maker.ts

```diff
@@ -1,70 +1,77 @@
 import erc20 from "./erc20";
 
-import networkIds from "./networks"
+import networkIds from "./networks";
 
 import makerProxyRegistryAbi from "./abi/makerdao/ProxyRegistry.json";
 import dssCdpManagerAbi from "./abi/makerdao/DssCdpManager.json";
 import dssProxyActionsAbi from "./abi/makerdao/DssProxyActions.json";
 
 const ilks = {
-  "BAT-A": {
+  bat_a: {
+    ilk: "BAT-A",
     token: { address: erc20.bat.address },
     join: {
       address: {
         [networkIds.mainnet]: "0x3D0B1912B66114d4096F48A8CEe3A56C231772cA"
-      },
+      }
     }
   },
-  "ETH-A": {
+  eth_a: {
+    ilk: "ETH-A",
```

Comment:
I think this should be called `symbol`. Ilk is just [defined](https://github.com/makerdao/dss/wiki/Glossary#cdp-engine---vat) as a "collateral type". The string "ETH-A" is like a ticker symbol on a stock market. At least that's my understanding.

```js
// this doesn't really make sense, there are 2 ilks in that sentence
maker.ilks.eth_a.ilk

// this makes more sense
maker.ilks.eth_a.symbol
```

Reply:
Sure